### DISCUSSION TITLE
Notification: "It seems that the maintenance isn't set up properly"?

### DISCUSSION THREAD

**Discussion by polymed** [pimcore/pimcore#8225] [Q&A / Support] [answered]
Hello folks,

although I have already set up a CronJob, I still get the notification "It seems that the maintenance isn't set up properly". 

Can anyone help me with this?

Best regards

**Comment by dvesh3** [comment, marked as answer]
@polymed setup cron job to run every hour  https://github.com/pimcore/pimcore/blob/1901f29f2aeda3321a5e0e1b8c45fa67b9c79199/bundles/AdminBundle/Controller/Admin/IndexController.php#L378-L380
